### ISSUE TITLE
Couldn't find tesseract dictionaries on Debian

### ISSUE THREAD

**Issue by sthibaul** [Nalin-x-Linux/lios-3#6]
Hello,

We've had issues with starting lios at all on Debian, because the tesseract data is in l"/usr/share/tesseract-ocr/4.00/tessdata" instead of "/usr/share/tesseract-ocr/tessdata". Perhaps the tesseract backend should just use the output of the "tesseract --list-langs" command?

Samuel

**Comment by svoboda77** [comment]
It's still not fixed in Debian 12 bookworm. To fix this in bookworm, create a softlink tessdata -> 5/tessdata in /usr/share/tesseract-ocr.

**Comment by sthibaul** [comment]
> It's still not fixed in Debian 12 bookworm. To fix this in bookworm, create a softlink tessdata -> 5/tessdata in /usr/share/tesseract-ocr.

That is not really a fix :)

lios should really use the `tesseract` command-line tool, instead of hardcoding paths

**Comment by Nalin-x-Linux** [comment]
I agree with @sthibaul. I think '--list-langs' can list the languages and we can remove other code since we are no longer support tesseract-training gui.
